common/app/views/support/cleaner/AmpAdCleaner.scala

```diff
@@ -87,14 +88,25 @@ object AmpAdCleaner {
 case class AmpAdCleaner(edition: Edition, uri: String, article: Article) extends HtmlCleaner {
 
   def adAfter(element: Element) = {
-    val ampAd = <div class="amp-ad-container">
-          <amp-ad width="300" height="250" type="doubleclick" data-loading-strategy="prefer-viewability-over-views"
-                  json={AmpAd(article, uri, edition.id.toLowerCase()).toString()}
-                  data-slot={AmpAdDataSlot(article).toString()}>
-          </amp-ad>
+    // data-use-beta-a4a-implementation will throw an error if the page is not in the AMP CDN
+    // Lets prevent it appearing in Dev
+    if(Configuration.environment.stage.toLowerCase == "dev") {
```

Comment:
There is a lot of duplication here, but it seems that conditionals can only be used to set the value of an attribute, not the presence of it. For example, this would work, but would still throw the error:

`data-use-beta-a4a-implementation={Configuration.environment.stage.toLowerCase != "dev"}`

If we aim to move the ampAd out into it's own template, it would simplify things.